tooling/nargo_fmt/tests/input/impl.nr

```diff
@@ -0,0 +1,11 @@
+impl Type {}
+
+impl Type<A> {}
+
+impl Type<A, B> {}
+
+impl Type {
+    fn type(self) {}
+
+    fn type(&mut self) {}
+}
```

Comment:
Can you throw in some incorrect/extra spaces and indentation in this test?
As-is it matches the expected result already.